lib/mixins/property-effects.html

```diff
@@ -411,32 +416,122 @@
         Object.assign(changedProps, inst.__dataPending);
         inputProps = inst.__dataPending;
         inst.__dataPending = null;
+        // Ensure all computed properties are de-duped against the same turn
+        dedupeId--;
       }
     }
   }
 
+  const TRANSITIVE_DEPENDENCY = '~transitive~dependency~';
```

Comment:
Can we use a Symbol here to make sure we are always unique? (however unlikely someone names a property exactly this)